src/Compilers/Core/Portable/Operations/UnaryOperatorKind.cs

```diff
@@ -41,6 +41,11 @@ public enum UnaryOperatorKind
         /// Represents the C# 'false' operator and VB 'IsFalse' operator.
         /// </summary>
         False = 0x6,
+
+        /// <summary>
+        /// Represents the C# '^' operator.
+        /// </summary>
+        Hat = 0x7,
```

Comment:
>Hat [](start = 8, length = 3)

Is this how it will be called in the language spec?

Reply:
Yup, we decided that in https://github.com/dotnet/csharplang/blob/6904c200e917967c5a06ebea7493b2dea848c4ab/meetings/2018/LDM-2018-09-05.md